website/pages/docs/plugins/sources/aws/configuration.md

```diff
@@ -101,6 +101,11 @@ This is the (nested) spec used by the AWS source plugin.
 
   In AWS organization mode, CloudQuery will source all accounts underneath automatically
 
+- `initialization_concurrency` (int) (default: 4)
```

Comment:
why such a low default if we have 50000 as the default for the syncing goroutines?

Reply:
50,000 is a number that is designed to be spread out over every single table being resolved... This is only for setting up the SDK (assuming a role and getting the enabled/disabled regions)... If a user needs a higher value they can always increase it

Reply:
Non blocking for this PR, but we could test by modifying the code to run on more accounts than we have in our playground.
In Azure I tested by creating 1800 duplicated subscriptions, see https://github.com/cloudquery/cloudquery/compare/main...erezrokah:test/azure_memory?expand=1#diff-c5caf1d71566c18f3cf9dfd107693cc1fca4ea980176d33db01d94b013685afeR226

The only catch is the need to keep client IDs unique

Reply:
So tested it with 7000 faked accounts... Only ran into rate limit issues when an account was duplicated... which should ever happen

Reply:
Nice! Does this mean we can use a higher default concurrency? If we're not sure we can keep it lower and. change the default later based on users reports